integrations/docs/Integration-API.md

```diff
@@ -0,0 +1,39 @@
+# Integration API
+
+Integrations are a stateful bundle which will be stored inside a system `.integration` index which will reflect the integration's status during the different phases of its lifecycle.
+
+---
+**Integration Loading Lifecycle API**: 
+
+## Load Integrations
+// TODO explain with details
+
+### Verify Integration
+// TODO explain with details
+
+### Create data-stream
+// TODO explain with details
+
+### Load visualizations
+// TODO explain with details
+
+### Load dashboards  
+// TODO explain with details
+
+### Load Sample data
```

Comment:
Should load sample data be after `verify integration`. Cause, visualizations and dashboards will depend on sample data.

Reply:
yes correct  - I'm adding the workflow document to detail this process